client/app/views/contacts/merge.coffee

```diff
@@ -11,8 +11,7 @@ module.exports = class MergeView extends Mn.ItemView
     attributes:
         role: 'alertdialog'
 
-    behaviors: ->
-        Navigator: {}
+    behaviors:
```

Comment:
@m4dz I think you should keep `->` here, to avoid sharing the same objects between two `MergeView` objects.